SplunkAppForWazuh/appserver/static/js/services/apiIndexStorageService/apiIndexStorageService.js

```diff
@@ -86,11 +86,8 @@ define(['../module'], function (app) {
             return parsedExtensions[apiId]
```

Comment:
Not every logical path will return a value. To write it consistently, it would be better if it had 1 return statement that will always be executed, even if the returned value is falsy one.

Reply:
The whole function has been refactored in the last commit